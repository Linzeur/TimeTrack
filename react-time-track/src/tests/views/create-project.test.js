import React from "react";
import { render } from "@testing-library/react";

import CreateProject from "../../views/create-project";

test("create-project component", () => {
  const { asFragment } = render(<CreateProject />);
  expect(asFragment()).toMatchSnapshot();
});
