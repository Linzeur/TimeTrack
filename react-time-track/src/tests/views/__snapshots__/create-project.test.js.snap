// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create-project component 1`] = `
<DocumentFragment>
  <section
    class="css-qvlzl1-InputInfoProject"
  >
    <form
      class="css-avrx2w-InputInfoProject"
    >
      <h2>
        Step 1: Enter general information
      </h2>
      <hr />
      <fieldset
        class="css-xeagov"
      >
        <label
          class="css-ooi7h"
        >
          Title
        </label>
        <div
          class="css-16q0q7d"
        >
          <input
            aria-label="Enter the Title of the project"
            class="css-pldvt4"
            name="txtTitle"
            required=""
            type="text"
            value=""
          />
        </div>
      </fieldset>
      <fieldset
        class="css-xeagov"
      >
        <label
          class="css-ooi7h"
        >
          Client
        </label>
        <div
          class="css-16q0q7d"
        >
          <input
            aria-label="Enter the Client of the project"
            class="css-pldvt4"
            name="txtClient"
            required=""
            type="text"
            value=""
          />
        </div>
      </fieldset>
      <fieldset
        class="css-xeagov"
      >
        <label
          class="css-ooi7h"
        >
          Category
        </label>
        <div
          class="css-16q0q7d"
        >
          <input
            aria-label="Enter the Category of the project"
            class="css-pldvt4"
            name="txtCategory"
            required=""
            type="text"
            value=""
          />
        </div>
      </fieldset>
      <fieldset
        class="css-xeagov"
      >
        <label
          class="css-ooi7h"
        >
          Product
        </label>
        <div
          class="css-16q0q7d"
        >
          <input
            aria-label="Enter the Product of the project"
            class="css-pldvt4"
            name="txtProduct"
            required=""
            type="text"
            value=""
          />
        </div>
      </fieldset>
      <fieldset
        class="css-xeagov"
      >
        <label
          class="css-ooi7h"
        >
          Start
        </label>
        <div
          class="css-16q0q7d"
        >
          <input
            aria-label="Enter the Start of the project"
            class="css-1ue9bta"
            name="txtStart"
            required=""
            type="date"
            value="2019-08-01"
          />
        </div>
      </fieldset>
      <fieldset
        class="css-xeagov"
      >
        <label
          class="css-ooi7h"
        >
          End
        </label>
        <div
          class="css-16q0q7d"
        >
          <input
            aria-label="Enter the End of the project"
            class="css-1ue9bta"
            name="txtEnd"
            required=""
            type="date"
            value="2019-08-01"
          />
        </div>
      </fieldset>
      <fieldset
        class="css-1pfy88f-InputInfoProject"
      >
        <button
          class="css-66se3v-InputInfoProject"
          type="button"
        >
          Cancel
        </button>
        <button
          class="css-66se3v-InputInfoProject"
          type="submit"
        >
          Next
        </button>
      </fieldset>
    </form>
  </section>
</DocumentFragment>
`;
